MODULE.bazel

```diff
@@ -0,0 +1,44 @@
+# This MODULE file can not be used for local development inside libsbp, due to problems
```

Comment:
I'm not so hot on bazel, a lot of this stuff is still in `WORKSPACE.bazel` as well. Does it need to be in both places?

Reply:
Yes, for now we keep the `WORKSPACE.bazel` file untouched to still fully support Bazel 6.
In case Bazel 7+bzlmod is selected, then the `MODULE.bazel` + `WORKSPACE.bzlmod` files are used.